src/admin-views/commerce/orders/single/order-details-metabox.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Single order - Details metabox.
+ *
```

Comment:
Missing @Since and @version

Reply:
@rafsuntaskin could you help me with the `@version` tag ? What are we using initially for a template file's version ?

Reply:
@dpanta94 the `@version` doc is for template files. It goes below the `@since` tag. Any time you create or update a template, the `@version` represents the current version.
```suggestion
 *
 * @since TBD
 *
 * @version TBD
```

Reply:
Unlike `@since` docs. There will only be one single `@version` doc.